aip/general/0191.md

```diff
@@ -105,11 +105,23 @@ When defining APIs, the following rules apply:
   - Package or namespace directives for other languages **must** be set either
     in every file in the proto package, or none of them. If they are set, the
     values **must** be identical in every file.
-
-**Important:** While other languages have sensible defaults, be aware that
-_adding_ this annotation (with a value not equivalent to the default)
-constitutes a breaking change in that language. When releasing protos, be sure
-that omissions are intentional.
+  - If any part of the protobuf package is a compound name (such as
+    accessapproval), C#, Ruby and PHP options **must** be specified in order
```

Comment:
We could potentially make this "should", but it really leads to a poor experience :(

Reply:
I can't think of a compelling reason to make this a "should", so I think keeping it a "must" makes sense.